roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -4,6 +4,10 @@ openshift_prometheus_state: present
 
 openshift_prometheus_namespace: openshift-metrics
 
+# defaults hosts for routes
+openshift_prometheus_prometheus_hostname: prometheus-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}
```

Comment:
I would change this to `openshift_prometheus_hostname`. This better aligns with other variable pattern